packages/language-server/src/plugins/typescript/features/FindReferencesProvider.ts

```diff
@@ -18,7 +18,7 @@ export class FindReferencesProviderImpl implements FindReferencesProvider {
         const { lang, tsDoc } = await this.getLSAndTSDoc(document);
         const fragment = tsDoc.getFragment();
 
-        const references = lang.getReferencesAtPosition(
```

Comment:
Don't know what's the difference between these two methods. But the one we used before no longer returns `isDefinition`

Reply:
From a quick TS codebase search it seems that `findReferences` is like `getReferencesAtPosition` with an additional step afterwards, at least they both call `getReferencedSymbolsForNode` which does the actual work

Reply:
Found a place where it does make a difference. If you use a go-to reference on the definition.
 It would invoke `includeDeclaration: true`. Previously if there is only one reference it would take you to the reference.  if we use the `findReferences` and remove the check it would not take you to the reference but show both reference and definition.